lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako

```diff
@@ -138,17 +138,12 @@
                 ## Help tab.
                 <%
                     menu_options = []
-                    qa_url = app.config.get( "qa_url", None )
-                    if qa_url:
-                        menu_options = [ [_('Galaxy Q&A'), qa_url, "_blank" ] ]
                     menu_options.extend( [
                         [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/toolshed" ), "_blank" ],
                         [_('Support'), app.config.get( "support_url", "https://galaxyproject.org/support" ), "_blank" ],
-                        [_('Search'), app.config.get( "search_url", "http://galaxyproject.org/search/" ), "_blank" ],
-                        [_('Mailing Lists'), app.config.get( "mailing_lists_url", "https://galaxyproject.org/mailing-lists" ), "_blank" ],
-                        [_('Videos'), app.config.get( "screencasts_url", "https://vimeo.com/galaxyproject" ), "_blank" ],
+                        [_('Videos'), app.config.get( "screencasts_url", "https://www.youtube.com/c/galaxyproject" ), "_blank" ],
                         [_('Wiki'), app.config.get( "wiki_url", "http://galaxyproject.org/" ), "_blank" ],
-                        [_('How to Cite Galaxy'), app.config.get( "citation_url", "https://galaxyproject.org/citing-galaxy" ), "_blank" ]
+                        [_('How to Cite Galaxy'), app.config.get( "citation_url", "https://academic.oup.com/nar/article/48/W1/W395/5849904" ), "_blank" ]
```

Comment:
This should probably be "How to cite the Tool Shed" ?

Reply:
Again, I was just trying to find any references to the things @nekrut wanted to change and change them. If it should be something else, then I'm happy to oblige.

Reply:
As mentioned above, this is part of the [Tool Shed](https://toolshed.g2.bx.psu.edu/)'s Help menu, so I think it makes more sense to mention the Tool Shed and link to either https://doi.org/10.1186/gb4161 or the "Citing Galaxy" Hub page (as before).